scripts/services/check-removed-services.js

```diff
@@ -0,0 +1,100 @@
+const { promises: fs } = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const servicesDir = path.resolve(__dirname, '../../services/');
+
+const checkRemovedServices = async (distFolder, servicesJSON) => {
+    // changes names of services to lower case without, and all - change to _
+    const normalizeFileName = (serviceName) => {
+        const specificCharacters = new RegExp(/[^a-z0-9.]/, 'g');
+        const lowerCased = serviceName.toLowerCase();
+        const replaceSpecialCharacters = lowerCased.replace(specificCharacters, '');
+        return replaceSpecialCharacters;
+    };
+    // get array with old services objects and check if json is valid
+    const getServicesData = async (servicesFile) => {
+        let OldServicesDataArray = [];
```

Comment:
should be camelCase
https://github.com/AdguardTeam/CodeGuidelines/blob/master/JavaScript/Javascript.md#naming--camelCase